src/NzbDrone.Core/MediaFiles/EpisodeImport/Manual/ManualImportService.cs

```diff
@@ -246,6 +246,7 @@ private List<ManualImportItem> ProcessFolder(string rootFolder, string baseFolde
 
                 if (files.Count() > 100)
                 {
+                    _logger.Warn("Found more than 100 files for multiple series. Skipping parsing.");
```

Comment:
It's possible they aren't for multiple series, they're just in a folder that doesn't have a series name so Sonarr doesn't bother parsing them all.